src/app/public/modules/selection-box/selection-box.module.ts

```diff
@@ -50,9 +53,7 @@ import {
     SkySelectionBoxHeaderComponent
   ],
   providers: [
-    MutationObserverService,
-    SkyCoreAdapterService,
-    SkyThemeService
+    SkySelectionBoxAdapterService
```

Comment:
☝️ Used by both the `SkySelectionBoxGridComponent` and `SkySelectionBoxComponent`.